src/query.rs

```diff
@@ -245,41 +287,22 @@ pub fn q_explain<'sqlite, 'schema, 'query, T>
  inputs: T) -> Result<QueryExplanation>
         where T: Into<Option<QueryInputs>>
 {
-    let parsed = parse_find_string(query)?;
-    let algebrized = algebrize_with_inputs(schema, parsed, 0, inputs.into().unwrap_or(QueryInputs::default()))?;
-
-    let unbound = algebrized.unbound_variables();
-    if !unbound.is_empty() {
-        bail!(ErrorKind::UnboundVariables(unbound.into_iter().map(|v| v.to_string()).collect()));
-    }
+    let algebrized = algebrize_query_str(schema, query, inputs)?;
 
     let empty_because = algebrized.cc.empty_because.clone();
```

Comment:
The string we generate for empty queries always has `LIMIT 0` but it's not always `SELECT 1 limit 0`, and the thing it selects seems like it could be generally useful for debugging. (Maybe not though, it's not like we're going to actually execute that string...)

Reply:
Aye, it's probably not useful; the different shape of the SELECT query will just be to match the expected caller.